sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,219 @@
+# May'21 release notes
+
+## New features
+ - [ESIMD] Allowed ESIMD and regular SYCL kernels to coexist in the same
+   translation unit and in the same program. The `-fsycl-explicit-simd` option
+   is no longer required for compiling ESIMD code and was deprecated. DPCPP RT
+   implicitly appends `-vc-codegen` compile option for ESIMD images.
+ - [ESIMD] Added indirect read and write methods to ESIMD class [8208427]
+ - Provided `sycl::ONEAPI::has_known_identity` type trait to determine if
+   reduction interface supports user-defined type at compile-time [0c7bd24]
+   [060fd50]
+ - Added support for multiple reduction items [c042f9e]
+ - Added support for `+=`, `*=`, `|=`, `^=`, `&=` operations for custom type
+   reducers [b249099]
+ - Added SYCL 2020 `sycl::kernel_bundle` support [5af118a] [dcfb6b1] [ae45333]
+   [8335e17]
+ - Added `sycl/sycl.hpp` entry header in compliance with SYCL 2020 [5edb228]
+   [24d179c]
+ - Added `__LIBSYCL_[MAJOR|MINOR|PATCH]_VERSION` macros, see
+   [PreprocessorMacros](doc/PreprocessorMacros.md) for more information
+   [9f3a74c]
+ - Added support for SYCL 2020 reductions with `read_write` access mode to
+   reduction variables [733d5e3]
+ - Added support for SYCL 2020 reductions with
+   `sycl::property::reduction::initialize_to_identity` property [3473c1a]
+ - Implemented zero argument version of `sycl::buffer::reinterpret()` for
+   SYCL 2020 [c0c3c80]
+ - Implemented [Matrix Programming Extension for DPC++](https://github.com/intel/llvm/blob/49b6749ea9175ae250b718c04d71af4ccfecc06c/sycl/doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc) [35db973]
```

Comment:
Is it full support or partial? @dkhaldi 
Written like a full support, but I'm not sure it is.

Reply:
@pvchupin The document does not say full support. Only base features are supported. Mainly the query interface is not there yet. Also, this is only AOT on CPU AMX.

I would suggest reword the release note to:
- Added an initial AOT implementation of the experimental matrix extension on the CPU device to target AMX hardware. Base features are supported.

Reply:
Sounds good to me. Thanks @dkhaldi for suggestion.